src/components/globals.css

```diff
@@ -53,14 +53,26 @@
 \*------------------------------------*/
 
 button {
-  all: unset;
+  border: none;
+  background: none;
+  text-align: left;
   cursor: pointer;
+  color: inherit;
+  font-size: inherit;
+  padding: 0;
+  margin: 0;
+  border-radius: inherit;
 }
 
 button:disabled {
   cursor: not-allowed;
 }
 
+button:focus {
+  outline-offset: 4px;
+  transition: none;
+}
```

Comment:
The `transition: none` here is breaking the animation in the close button for our burger menu